docs/api/time.rst

```diff
@@ -0,0 +1,18 @@
+
+.. xpy:module:: datatable.time
+
+datatable.time
+==============
+
+.. list-table::
+   :widths: auto
+   :class: api-table
+
+   * - :func:`ymd(y,m,d)`
+     - Create a ``date32`` column from year, month, day components.
```

Comment:
"year, month, day" -> "year, month and day"?

Reply:
ok